src/visualization/types/Band/index.tsx

```diff
@@ -1,15 +1,13 @@
-import icon from './icon'
-import properties from './properties'
-import options from './options'
-import view from './view'
+import {BandIcon} from './icon'
+import {BandOptions} from './options'
+import {BandProperties} from './properties'
+import {Band} from './view'
 
-export default register => {
-  register({
-    type: 'band',
-    name: 'Band',
-    graphic: icon,
-    initial: properties,
-    component: view,
-    options,
-  })
+export const view = {
```

Comment:
what happens if we did something like:

```ts
export const view implements Visualization = {
```

does that work? I think this might be a missing piece of this puzzle

Reply:
>This works but the `view` becomes a JavaScript `class` (not to be confused with a React class component). I'll push up the changes.

@TCL735 what do you think of this change? Is it worth it? Too much overhead to just `implement` an interface?

Reply:
It's fine to have JavaScript classes. The ones we are implementing are small and very straightforward. The benefit we get is an easy-to-follow type for future reference in making new visualizations.